conf/report-template.html

```diff
@@ -338,6 +347,7 @@
     <table id="report_table">
       <thead>
         <tr>
+          <th>  </th>
```

Comment:
Maybe we should add the headers here? It looks weird if the first two columns don't have any headers